roles/swlib/tasks/gcsdirect.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
+# Copyright 2025 Google LLC
```

Comment:
nit: no need to update this for existing files: https://opensource.google/documentation/reference/copyright

Reply:
No worries, this is a small nit anyway.